src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java

```diff
@@ -74,42 +73,42 @@ public static void checkUser(Request req) {
                     LOG.info("New user is creating self. OK to proceed without existing user object for auth0UserId");
                 } else {
                     // Otherwise, if no valid user is found, halt the request.
-                    logMessageAndHalt(req, HttpStatus.NOT_FOUND_404, "Unknown user.");
+                    JsonUtils.logMessageAndHalt(req, HttpStatus.NOT_FOUND_404, "Auth0 auth - Unknown user.");
```

Comment:
This message could be improved. It's hard to read as it stands.

Reply:
Updated.